include/swift/SILOptimizer/Utils/CanonicalOSSALifetime.h

```diff
@@ -313,6 +313,23 @@ class CanonicalizeOSSALifetime {
   CanonicalOSSAConsumeInfo consumes;
 
 public:
+  /// When rewriting destroys, is this an instruction which destroys should not
+  /// be hoisted over to avoid churn and infinite looping.
+  static bool mustNotHoistDestroyOver(SILInstructionKind kind) {
```

Comment:
`mustNotHoistDestroyOver` will be misinterpreted as "intructions that destroy must not be hoisted over". That's wrong. Instead these are "instructions that themselves are not worth hoisting a destroy over".

So `ignoredByDestroyHoisting` seems closer to the mark.

Reply:
Fixed.